hnn_core/gui/_viz_manager.py

```diff
@@ -238,7 +287,41 @@ def _plot_on_axes(b, widgets_simulation, widgets_plot_type,
         "spectrogram_cm": spectrogram_colormap_selection.value
     }
 
-    _update_ax(fig, ax, single_simulation, sim_name, plot_type, plot_config)
+    dpls_processed = _update_ax(fig, ax, single_simulation, sim_name,
+                                plot_type, plot_config)
+
+    # If target_simulations is not None and we are plotting a dipole,
+    # we need to plot the target dipole as well.
+    if target_simulations.value in data['simulations'].keys(
+    ) and plot_type == 'current dipole':
+
+        target_sim_name = target_simulations.value
+        target_sim = data['simulations'][target_sim_name]
+        # plot the target dipole.
+        target_plot_config = copy.deepcopy(plot_config)
```

Comment:
is the copy necessary?

Reply:
yeah this might be redundant, just habitually copied a dict...